include/swift/AST/Decl.h

```diff
@@ -613,8 +613,9 @@ class alignas(1 << DeclAlignInBits) Decl {
     unsigned : NumDeclBits;
 
     unsigned NumberOfVTableEntries : 2;
+    unsigned NumberOfFieldOffsetVectorEntries : 2;
```

Comment:
Is this ever more than one?

Reply:
Shouldn't be. Maybe I'm irrationally afraid of one-bit bitfields.

Reply:
Unsigned ones are safe!